code/modules/spells/spell_types/pointed/terrorize.dm

```diff
@@ -0,0 +1,40 @@
+/datum/action/cooldown/spell/pointed/terrorize
+	name = "Terrorize"
+	desc = "Stare down a victim with your piercing red eyes, inflicting them with terror buildup. \
+		Prey will become increasingly terrified. Swatting terrified prey with an open hand will \
+		scare and disorient them."
+	button_icon_state = "terrify"
+	background_icon_state = "bg_alien"
+	overlay_icon_state = "bg_alien_border"
+	panel = null
+	spell_requirements = NONE
+	cooldown_time = 25 SECONDS
+	cast_range = 9
+	active_msg = "You prepare to stare down a target..."
+	deactive_msg = "You refocus your eyes..."
+
+/datum/action/cooldown/spell/pointed/terrorize/is_valid_target(atom/cast_on)
+	. = ..()
+
+	if(!ishuman(cast_on))
+		to_chat(owner, span_notice("[cast_on] cannot be terrorized!"))
```

Comment:
might be nice to have this be a balloon alert like "can't be terrorized!"? seems more useful to me